Code/Editor/TrackView/TrackViewDialog.cpp

```diff
@@ -883,31 +894,47 @@ void CTrackViewDialog::Update()
     }
 
     // Display the name of the active camera in the static control, if any.
-    // The active camera node means two conditions:
-    // 1. Sequence camera is currently active.
-    // 2. The camera which owns this node has been set as the current camera by the director node.
-    IMovieSystem* movieSystem = AZ::Interface<IMovieSystem>::Get();
+    // Having an active camera means at least the following conditions:
+    // 1. The movie system has the Animation Sequence corresponding to the active TrackView Sequence.
+    // 2. The Animation Sequence has an active Director node.
+    // 3. The current Camera parameters in the movie system are valid.
+    // TODO: invalidate the Camera parameters in the movie system when conditions 1 and 2 are not valid.
```

Comment:
if you have a github issue for this todo, please add it in this comment

Reply:
AFAIK no GHI was created, but I believe that @l-sempe started looking into refactoring MovieSystem and copy / pasted code in Maestro and LyShine